l1-contracts/src/governance/Apella.sol

```diff
@@ -37,6 +37,10 @@ contract Apella is IApella {
     gerousia = _gerousia;
 
     configuration = DataStructures.Configuration({
+      proposeConfig: DataStructures.ProposeConfiguration({
+        lockDelay: Timestamp.wrap(3600),
```

Comment:
make an issue to update the delays here

Reply:
https://github.com/AztecProtocol/aztec-packages/issues/9750